lib/cartopy/sphinxext/summarise_package.py

```diff
@@ -62,13 +64,15 @@ def walk_module(mod_name, exclude_folders=None):
         files.sort()
         folders.sort()
 
-        def is_py_src(fname):
-            root, ext = os.path.splitext(fname)
-            return ext in ('.py', '.so')
-
-        files = filter(is_py_src, files)
-
         for fname in files:
+            basename, ext = os.path.splitext(fname)
+            if ext == '.so':
+                basename, soabi = os.path.splitext(basename)
+                if soabi.lstrip('.') != SOABI:
+                    continue
```

Comment:
I'm not able to test this as I am on linux and I have
```
>>> import sysconfig
>>> print sysconfig.get_config_var('SOABI')
None
```

but it looks like it is doing the correct thing with filenames that are like 
`_crs.cpython-32mu.so` when you have:
```
>>> sysconfig.get_config_var('SOABI')
'cpython-32mu'
```